### PR TITLE
build(node): Upgrade Node to latest LTS: 10.16.3

### PR DESCRIPTION
**Pull request by BYK**
See https://github.com/getsentry/docker-sentry/pull/192#discussion_r316384748.

### CODE REVIEW

**Comment by BYK** [comment]
`zeus` fails with 404 errors, not sure if that's expected or related to this change at all:

```
$ $(./bin/yarn global bin)/zeus upload -t "application/webpack-stats+json" .artifacts/*webpack-stats.json

error:   Artifact upload failed: 404 NOT FOUND
```

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-8.15.1
+10.16.3
```
Comment (markstory) [inline .nvmrc]: We should also bump the version in `package.json` for volta so that development environments are consistent.

Comment (BYK) [reply .nvmrc]: Ugh, I was already getting tired of the spread I've been tidying up. Incoming fix...

File: package.json [hunk 2]
```diff
@@ -178,7 +178,7 @@
     "webpack-profile": "yarn -s webpack --profile --json > stats.json"
   },
   "volta": {
-    "node": "8.15.1",
-    "yarn": "1.16.0"
+    "node": "10.16.3",
+    "yarn": "1.17.3"
```
Comment (billyvg) [inline package.json]: Is our checked in yarn @ 1.17.3?

**Review by billyvg — APPROVED** [review]
Can we deploy getsentry to staging first? Otherwise LGTM

File: package.json (hunk 2 above)
Comment (BYK) [reply package.json]: Yup

**Pull request merged**